modules/shadertools/src/modules/lighting/lighting.glsl.js

```diff
@@ -1,18 +1,16 @@
 export default `\
 struct AmbientLight {
  vec3 color;
- float intensity;
 };
 
 struct PointLight {
  vec3 color;
- float intensity;
  vec3 position;
+ vec3 attenuationConstLinExp;
```

Comment:
shorter name?

Reply:
You suggest something? It's Constant-Linear-Exponential

Reply:
attenuationExp

Reply:
> shorter name?

Most importantly, please don't use abbreviated words. Wouldn't `attenuationConstants` work just fine?